eloquent/distribution.yaml

```diff
@@ -908,6 +911,7 @@ repositories:
       version: 1.0.0-1
     source:
       test_abi: true
+      test_pull_requests: true
```

Comment:
This isn't set for Dashing. Why should it be for Eloquent?

Reply:
This was turned on based on @nuclearsandwich 's comment above:

> The *_vendor packages were turned off at one point because they were either deemed "trivial" or didn't install correctly enough to work in a devel job. But they're all passing in Eloquent so I would elect to keep them.